src/hotspot/share/classfile/systemDictionary.cpp

```diff
@@ -1292,7 +1292,9 @@ InstanceKlass* SystemDictionary::load_instance_class_impl(Symbol* class_name, Ha
 
     // find_or_define_instance_class may return a different InstanceKlass
     if (k != NULL) {
+#if INCLUDE_CDS
       SharedClassLoadingMark slm(THREAD, k);
+#endif
```

Comment:
I think it's cleaner to use the CDS_ONLY macro. This macro is already used elsewhere in this file.

```
CDS_ONLY(SharedClassLoadingMark slm(THREAD, k));
```